src/service/viewport/viewport-binding-ios-embed-wrapper.js

```diff
@@ -112,6 +120,10 @@ export class ViewportBindingIosEmbedWrapper_ {
       get: () => body,
     });
 
+    if (this.useLayers_) {
+      installLayersServiceForDoc(this.ampdoc, this.wrapper_);
```

Comment:
Ditto, if we do the other way around and just create the layer with the `viewport` instance in the constructor, things could be a bit cleaner as well, imo.

Reply:
Are you suggesting that we `installLayersServiceForDoc(this)` (`this` being the `Viewport` or `ViewportBinding`)?